frontend/lib/norent/components/logo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { BulmaImageClass } from "../../ui/bulma";
+import { StaticImage } from "../../ui/static-image";
+import { getImageSrc } from "../homepage";
+
+type NorentLogoColor = "white" | null;
+
+export const NorentLogo = (props: {
+  size: BulmaImageClass;
+  color?: NorentLogoColor;
+}) => (
+  <div className="jf-norent-logo">
+    <StaticImage
+      ratio={props.size}
+      src={getImageSrc(props.color === "white" ? "logo-white" : "logo")}
```

Comment:
I know there is a way to dynamically set the color of an svg but I suspected it would require some refactoring of the svg files themselves... also since we only need two colors of the logo currently, I didn't spend too much time investigating.

Reply:
Ah cool that's fine, maybe add a comment in the code mentioning this?  It could just be a copy/paste of what you wrote here but up to you.